nodes/widgets/ui_table.html

```diff
@@ -130,6 +138,14 @@
         <label for="node-input-maxrows"><i class="fa fa-tag"></i> <span data-i18n="ui-table.label.maxRows"></label>
         <input type="number" id="node-input-maxrows">
     </div>
+    <div class="form-row">
+        <label for="node-input-selectionType"><i class="fa fa-crosshairs"></i> <span data-i18n="ui-table.label.selection"></span></label>
+        <select id="node-input-selectionType">
+            <option value="none" data-i18n="ui-table.selection.none"></option>
+            <option value="checkbox" data-i18n="ui-table.selection.checkbox"></option>
+            <option value="click" data-i18n="ui-table.selection.click"></option>
```

Comment:
Should single come before multiple like it does in the docs?

```suggestion
            <option value="none" data-i18n="ui-table.selection.none"></option>
            <option value="click" data-i18n="ui-table.selection.click"></option>
            <option value="checkbox" data-i18n="ui-table.selection.checkbox"></option>
```